Author: Riku Voipio <riku.voipio@linaro.org>
Description: fuse_kernel.h: clean includes
 Use <linux/types.h> for linux and define types used for other operating systems
 using <stdint.h> types (Closes: #752081).

diff -Naurp fuse.orig/include/fuse_kernel.h fuse/include/fuse_kernel.h
--- fuse.orig/include/fuse_kernel.h
+++ fuse/include/fuse_kernel.h
@@ -88,12 +88,16 @@
 #ifndef _LINUX_FUSE_H
 #define _LINUX_FUSE_H
 
-#include <sys/types.h>
+#ifdef __linux__
+#include <linux/types.h>
+#else
+#include <stdint.h>
 #define __u64 uint64_t
 #define __s64 int64_t
 #define __u32 uint32_t
 #define __s32 int32_t
 #define __u16 uint16_t
+#endif
 
 /*
  * Version negotiation:
